src/patterns/Locale/Locale.vue

```diff
@@ -78,6 +87,10 @@
           text: locale.nativeName
         }))
       }
+    },
+    created(){
+      this.$i18n.locale = localStorage.getItem('locale') || this.language
     }
+    
```

Comment:
We do not leave newlines at the start and the end of code blocks demarcated by `{` and `}`.